### PR TITLE
Don't use aliases nor http-last-modified to set the feed's last modified

### LINKED ISSUE #261: feed update problem
everything is in here: https://github.com/nextcloud/news/issues/607

### PR DESCRIPTION
**Pull request by alexdebril**
fix #261

### CODE REVIEW

**Comment by kesselb** [comment]
That will break http://blog.fefe.de/rss.xml

File: src/FeedIo/Standard/Rss.php [hunk 1]
```diff
@@ -106,6 +107,7 @@ public function buildItemRuleSet() : RuleSet
         $ruleSet
             ->add(new Author(), ['dc:creator'])
             ->add(new PublicId())
+            ->add($this->getModifiedSinceRule(static::DATE_NODE_TAGNAME), ['lastBuildDate', 'lastPubDate'])
```
Comment (kesselb) [inline src/FeedIo/Standard/Rss.php]: `lastBuildDate` or `lastPubDate` are not available for items.

**Comment by kesselb** [comment]
I guess if the HttpLastModified fixer runs after LastModified it should work for both feeds. Probably the LastModified fixer return null if there are no items with a date found or the HttpLastModified fixer run if the Date ist @0. But I don't have time this week to test it further :see_no_evil:

File: src/FeedIo/Standard/Rss.php (hunk 1 above)
Comment (alexdebril) [reply src/FeedIo/Standard/Rss.php]: They're not supposed to be, however I have already seen feeds with one of them in the Item node.

**Comment by alexdebril** [comment]
Thanks for providing your feedback so fast, you prevented another hotfix release! 
Your suggestion is the right thing to do: first trigger the LastModified fixer based on items and then the HttpLastModified. It will fetch the header only if there's no date at all in the feed

**Pull request merged**